### DISCUSSION TITLE
Organization repository clone error

### DISCUSSION THREAD

**Discussion by ghost** [coder/code-server#4657] [Q&A]
I use google translate because I don't speak english.

I'm trying to clone a github repository using code-server. Public and private repositories for personal accounts and public repositories for organizational accounts can be cloned successfully, but only "private repositories for organizational accounts" cannot be cloned.

```abc@0db36bf6ba73:~/workspace$ git clone https://github.com/FishNetwork/OfficialShop.git```
```Cloning into 'OfficialShop'...```
```remote: Repository not found.```
```fatal: repository 'https://github.com/FishNetwork/OfficialShop.git/' not found```

Please tell me how to resolve

**Comment by lg-nineoneninethreenine** [comment]
Since it's a Git repo, can you try using the GitHub CLI for cloning? I think that'll work.

**Reply by jsjoeio** [reply]
Did this end up working? It could also be if it's a private repository and you don't have the right git credentials setup locally.

**Reply by lg-nineoneninethreenine** [reply]
I kinda feel that might be irrelevant in the case of GitHub CLI(available on https://cli.GitHub.com) since the client configures Git once you authenticate it.

**Reply by jsjoeio** [reply]
Oh yeah true. I guess we'll have to wait to hear back to see if they fixed their issue

**Reply by lg-nineoneninethreenine** [reply]
Maybe it might also be org access. I think GitHub documented this for their CLI, need to do some sleuthing to find that webpage.

**Reply by ghost** [reply]
This has been resolved
Apparently it didn't allow permission to the organizational repository

**Reply by lg-nineoneninethreenine** [reply]
Okay, great you solved it!
